e2e-tests/README.md

```diff
@@ -48,34 +48,35 @@ ___Running tests on SauceLabs___
 
 To use SauceLabs instead of an instance of Firefox running locally, do the following:
 
-Create a text file called `.saucelabs` and put it in the e2e-tests directory. Get a username and key for SauceLabs and then add the following details to the `.saucelabs` file:
+Create a text file called `.saucelabs` and put it in your home directory. Get a
```

Comment:
It's best for this to be outside of the repository so it's not accidentally included in a pull request.